app/views/freelancers/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div>
+  <%= image_tag(@freelancer.image, size: "100x100", class: "image-fl-index", id: "image_#{@freelancer.id}") %>
+  <h3>Name: <%= @freelancer.name %></h3>
+  <p><%= @freelancer.description %></p>
+  <p>$<%= @freelancer.price %>/hr</p>
+  <%= button_to "Choose Me!", cart_index_path(freelancer_id: @freelancer.id), class: "btn btn-danger"%>
```

Comment:
our wireframe had categories listed on it for this show page.

we don't have the many to many set up between freelancers, categories, and the table that links these, so we'll need to create that in the future.